x-pack/test/reporting/api/chromium_tests.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function ({ loadTestFile, getService }) {
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('chromium', () => {
+    before(async () => {
+      await esArchiver.load('../../../../test/functional/fixtures/es_archiver/dashboard/current/kibana');
```

Comment:
nit: could assign `../../../../test/functional/fixtures/es_archiver/dashboard/current` to a variable and use a template literal.